gems/sorbet-runtime/lib/types/private/methods/call_validation_2_6.rb

```diff
@@ -73,7 +81,7 @@ def self.create_validator_method_fast0(mod, original_method, method_sig, return_
             message,
             'Return value',
             nil,
-            return_type,
+            method_sig.return_type,
```

Comment:
This method appeared to be the only one using `return_type` here instead of `method_sig.return_type`, which appears to be insignificant.

Yay for code generation! The code is now more consistent.